src/tests/sqlsmith/src/lib.rs

```diff
@@ -365,6 +380,8 @@ impl<'a, R: Rng> SqlGenerator<'a, R> {
                 from.push(self.gen_from_relation());
             }
         }
+        self.merge_parallel_to_relation();
```

Comment:
Seems llke we call `self.merge_parallel_to_relation` twice?

Reply:
Yes, Because the  gen_from have two return function, we need to make sure merge is called before return.